### PR TITLE
tests: add more tests for async comments

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
The second commit removes an unused file. I checked all our projects, but couldn't find any usage. (The image services were used in opin, so there are services from a4 used somewhere...)

### CODE REVIEW

File: tests/comments/test_async_templatetags.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.contenttypes.models import ContentType
+from django.urls import reverse
+from freezegun import freeze_time
+
+from adhocracy4.test import helpers
+from tests.apps.questions.phases import AskPhase
+
+
+def react_comment_render_for_props(rf, user, question):
```
Comment (Rineee) [inline tests/comments/test_async_templatetags.py:15]: I was wondering that before: where does this rf fixture come from and what does it stand for actually? :sweat_smile:

**Review by Rineee — APPROVED** [review]
So supercool!! And we are way over 90% now, wohooooo :partying_face:

File: tests/comments/test_async_templatetags.py (hunk 1 above)
Comment (fuzzylogic2000) [reply tests/comments/test_async_templatetags.py:15]: That is a very good question! I think it stands for RequestFactory, but I have no idea how it gets into our tests as rf.
https://docs.djangoproject.com/en/3.2/topics/testing/advanced/#the-request-factory
@Rineee

Comment (Rineee) [reply tests/comments/test_async_templatetags.py:15]: Ah cool, thanks! Now that I knew what to look for, I investigated and it comes with pytest, see here: https://github.com/pytest-dev/pytest-django/blob/6ac1495c17d94db1293659edb01966c278064114/pytest_django/fixtures.py#L378

Comment (fuzzylogic2000) [reply tests/comments/test_async_templatetags.py:15]: Nice! Thanks!

**Pull request merged**